src/interaction/inspector.jl

```diff
@@ -30,28 +35,29 @@ function bbox2string(bbox::Rect2)
          x: (%0.3f, %0.3f)
          y: (%0.3f, %0.3f)
         """,
-        p0[1], p1[1], p0[2], p1[2]
+        p0[1],
+        p1[1],
+        p0[2],
+        p1[2]
     )
 end
 
 color2text(c::AbstractFloat) = @sprintf("%0.3f", c)
 color2text(c::Symbol) = string(c)
 color2text(c) = color2text(to_color(c))
-function color2text(c::RGBAf)
+color2text(c::RGBAf) =
     if c.alpha == 1.0
         @sprintf("RGB(%0.2f, %0.2f, %0.2f)", c.r, c.g, c.b)
     else
         @sprintf("RGBA(%0.2f, %0.2f, %0.2f, %0.2f)", c.r, c.g, c.b, c.alpha)
     end
-end
 
 color2text(name, i::Integer, j::Integer, c) = "$name[$i, $j] = $(color2text(c))"
 function color2text(name, i, j, c)
     idxs = @sprintf("%0.2f, %0.2f", i, j)
     "$name[$idxs] = $(color2text(c))"
```

Comment:
How come this didn't get a `return`?

Reply:
The `"$name[$idxs] = $(color2text(c))"`

Reply:
I've disabled `always_use_return = true` on purpose.
IMO this is pretty minor, and I think we should make the formatter the least intrusive as possible.

Reply:
Huh. I saw a bunch of other places get a return, so I got confused why there wasn't one here. Maybe I was looking at different commits.

Reply:
Yeah, sorry about the changes, I'm trying to propose something not too disruptive for a start, which could maybe be polished later ...